baml_language/crates/bridge_go/build.sh

```
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+PROTO_DIR="${SCRIPT_DIR}/../bridge_ctypes/types"
+OUT_DIR="${SCRIPT_DIR}/cffi/proto"
+
+mkdir -p "${OUT_DIR}"
+
+# Resolve protoc-gen-go
+if [ -n "${PROTOC_GEN_GO_PATH:-}" ]; then
+    export PATH="$(dirname "${PROTOC_GEN_GO_PATH}"):${PATH}"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)